views/keyword/index.html

```diff
@@ -1,16 +1,27 @@
-<div class="card-keyword">
-  <div class="card-keyword__content">
-    <h1 class="card-keyword__title">{{ .Title }}</h1>
+<div class="keyword-main">
+  <h1 class="keyword-main__title">{{ .Title }}</h1>
+  <div class="keyword-main__content">
     <form
-      class="form-keyword form"
+      class="form keyword-upload-form"
```

Comment:
The class name should be:

```suggestion
      class="form form-keyword-upload"
```

From [Compass](https://nimblehq.co/compass/development/code-conventions/css/):

> Use the UI element type as a prefix following by one to two words.